src/main/java/com/hackclub/hccore/commands/LocCommand.java

```diff
@@ -125,6 +142,10 @@ public boolean onCommand(CommandSender sender, Command cmd, String alias, String
             default:
                 return false;
         }
+        for (String var : args) {
```

Comment:
Remove the debug code

Reply:
oops. as i said this is really a draft pr.